src/styles/main.scss

```diff
@@ -0,0 +1,135 @@
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin-top: 50px;
+  padding: 50px, 40px;
+  margin-bottom: 72px;
+  gap: 48px;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+}
+
+.card {
+  transition-duration: 300ms;
+  width: 200px;
+  height: 400px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500px;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    color: #616070;
+    font-weight: 400px;
```

Comment:
The `font-weight` property should not have a unit. It should be `font-weight: 400;` instead of `400px`.